CHANGELOG.md

```diff
@@ -9,12 +9,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- Tarantool 3.0 role for expirationd.
```

Comment:
Please, add links to the issue in the changelog entries.

Reply:
Changed